pages/common/git-rebase.md

```diff
@@ -4,9 +4,9 @@
 > Commonly used to "move" an entire branch to another base, creating copies of the commits in the new location.
 > More information: <https://git-scm.com/docs/git-rebase>.
 
-- Rebase the current branch on top of the master branch:
+- Rebase the current branch on top of the another specified branch:
 
-`git rebase {{master}}`
+`git rebase {{branch_name}}`
```

Comment:
As mentioned above, 

> It's unclear whether `branch_name` refers to the current branch or the target branch. We might want to use a more explicit token name here.

Reply:
```suggestion
`git rebase {{base_branch}}`
```

Reply:
I'll have to quote myself again (the two discussions are the same, only in two different languages):

> I'd go with `target_branch` or `new_base_branch`. That's because the very notion of rebasing implies two bases, the old one and the new one, so merely `base_branch` would still be ambiguous as it could refer to either base.

Reply:
Reading [the docs](https://git-scm.com/book/en/v2/Git-Branching-Rebasing#_more_interesting_rebases:~:text=More%20Interesting%20Rebases), in `More Interesting Rebases` section, they use the term:
```suggestion
`git rebase {{rebase_target_branch}}`
```

Reply:
Great point, @waldyrious. `new_base_branch` sounds the most clear to me, so how about we go with that?

Reply:
Applied `new_base_branch`, so marking this as resolved. Feel free to unresolve if this isn't the case.